packages/validator/src/commands/convertCmd.ts

```diff
@@ -0,0 +1,49 @@
+import { convert, convertJson } from '../convertor';
+import { Log, Result } from '../types';
+import * as utils from './utils';
+import fs from 'fs';
+import { invalidConversionMessage } from '../utils/messages';
+
+const oas2ois = 'OAS2OIS.json';
+const ois2config = 'OIS2Config.json';
+
+if (process.env.npm_config_template) {
+  console.log(
+    JSON.stringify(
+      convert(process.env.npm_config_specs || process.argv[3], process.env.npm_config_template),
+      null,
+      '\t'
+    )
+  );
+} else {
+  const from = (process.env.npm_config_from || process.argv[2]).toLowerCase();
+  const to = (process.env.npm_config_to || process.argv[3]).toLowerCase();
+  const specs = process.env.npm_config_specs || process.argv[4];
+  const version = process.env.npm_config_fromVersion || process.argv[5];
```

Comment:
Is it expected that the user *either* supply environment variables *or* commandline variables? If so, I find this a bit confusing because it seems like the user can do a mix of both.

Reply:
Validator can process mix of both variables, it is a bit of useless feature, but I don't see a reason why not to allow it.